skyportal/handlers/api/source.py

```diff
@@ -257,6 +257,7 @@ def get_source(
     source_info["luminosity_distance"] = s.luminosity_distance
     source_info["dm"] = s.dm
     source_info["angular_diameter_distance"] = s.angular_diameter_distance
+    source_info["ebminusv"] = s.ebminusv
```

Comment:
I used to call this field `ebv` and it seemed to be fairly well understood. Also "extinctions" may be a good name.

Reply:
ebv is nice.